src/test/java/org/opensearch/security/auditlog/config/AuditConfigSerializeTest.java

```diff
@@ -13,18 +13,19 @@
  *  permissions and limitations under the License.
  */
 
-package com.amazon.opendistroforelasticsearch.security.auditlog.config;
+package org.opensearch.security.auditlog.config;
 
-import com.amazon.opendistroforelasticsearch.security.auditlog.impl.AuditCategory;
-import com.amazon.opendistroforelasticsearch.security.compliance.ComplianceConfig;
-import com.amazon.opendistroforelasticsearch.security.support.ConfigConstants;
-import com.amazon.opendistroforelasticsearch.security.support.WildcardMatcher;
+import org.opensearch.security.auditlog.impl.AuditCategory;
+import org.opensearch.security.compliance.ComplianceConfig;
+import org.opensearch.security.support.ConfigConstants;
+import org.opensearch.security.support.WildcardMatcher;
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.InjectableValues;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.node.ObjectNode;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableSet;
+import org.junit.Assert;
```

Comment:
Revert this and line 39, 40, 184, 277, 278, 330, 331

Reply:
Done